backend/Repositories/SightingRepo.cs

```diff
@@ -24,8 +25,17 @@ public SightingRepo(WhaleSpottingDbContext context)
         public IEnumerable<Sighting> GetAllSightings()
         {
             return _context
-                .Sightings
-                .Include(s => s.Species);
+                    .Sightings
+                    .Include(s => s.Species);
+        }
+
+        public Sighting GetSightingById(int id)
+        {
+            return _context
+                    .Sightings
+                    .Where(s => s.Id == id)
+                    .Include(s => s.Species)
+                    .Single();
```

Comment:
You can make this slightly more efficient by doing the `Single` first:
```suggestion
                .Single(s => s.Id == id)
                .Include(s => s.Species);
```